### PR TITLE
virt: Adjust wording

### PR DESCRIPTION
**Pull request by fabiand**
This change is adjusting the wording in order to make the plugin discoverable
and alignt he language.

Signed-off-by: Fabian Deutsch <fabiand@fedoraproject.org>

### CODE REVIEW

File: plugins/virt.yaml [hunk 1]
```diff
@@ -35,11 +35,11 @@ spec:
         - from: "/virtctl/virtctl-v0.17.2-darwin-amd64"
           to: "virtctl"
       bin: "virtctl"
-  shortDescription: controls virtual machine related operations.
+  shortDescription: Control KubeVirt Virtual Machines
```
Comment (corneliusweig) [inline plugins/virt.yaml]: I understand the camel-case for "KubeVirt", but "virtual machines" is just plain English and can be lower-case, right? Also, I think the short description could hint that this is a wrapper around `virtctl`
```suggestion
  shortDescription: Control KubeVirt virtual machines using virtctl
```

Comment (fabiand) [reply plugins/virt.yaml]: Both good points, I'll address them. Thanks.

Comment (fabiand) [reply plugins/virt.yaml]: (Added a new commit as it suites my flow better)

**Comment by fabiand** [comment]
Thanks!

**Pull request merged**